CMakeLists.txt

```diff
@@ -332,52 +315,55 @@ Configuration:
 ")
 
 option(CMAKE_DEBUG_INCLUDES_LDFLAGS "List include dirs and ldflags for xournalpp target" OFF)
-mark_as_advanced (FORCE CMAKE_DEBUG_INCLUDES_LDFLAGS)
+mark_as_advanced(FORCE CMAKE_DEBUG_INCLUDES_LDFLAGS)
 if (CMAKE_DEBUG_INCLUDES_LDFLAGS)
-	message("Include directories: ${xournalpp_INCLUDE_DIRS}")
-	message("LDFLAGS/LIBRARIES: ${xournalpp_LDFLAGS}")
+    message("Include directories: ${xournalpp_INCLUDE_DIRS}")
+    message("LDFLAGS/LIBRARIES: ${xournalpp_external_deps}")
 endif (CMAKE_DEBUG_INCLUDES_LDFLAGS)
 
-
+##############################
 # Packaging options
-set (CPACK_OUTPUT_FILE_PREFIX packages)
-set (CPACK_PACKAGE_NAME "${PROJECT_PACKAGE}")
-set (CPACK_PACKAGE_DESCRIPTION_SUMMARY "Xournal++ - Open source hand note-taking program")
-set (CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/package_description")
-set (CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE")
-set (CPACK_PACKAGE_INSTALL_DIRECTORY "Xournal++")
-set (CPACK_PACKAGE_FILE_NAME "xournalpp-${PROJECT_VERSION}-${DISTRO_NAME}-${DISTRO_CODENAME}-${PACKAGE_ARCH}")
-set (CPACK_STRIP_FILES ON)  # Debian packages must be stripped from debug messages, so lintian doesn't complain
+# !!! only override defaults if necessary !!!
+##############################
+
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION}~git${PACKAGE_TIMESTAMP}-${RELEASE_IDENTIFIER}-${DISTRO_CODENAME})
+set(CPACK_OUTPUT_FILE_PREFIX packages)
+set(CPACK_PACKAGE_NAME "${PROJECT_NAME}")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Xournal++ - Open source hand note-taking program")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/package_description")
+set(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE")
+set(CPACK_PACKAGE_INSTALL_DIRECTORY "Xournal++")
+set(CPACK_PACKAGE_FILE_NAME "xournalpp-${PROJECT_VERSION}-${DISTRO_NAME}-${DISTRO_CODENAME}-${PACKAGE_ARCH}")
+set(CPACK_STRIP_FILES ON) # Debian packages must be stripped from debug messages, so lintian doesn't complain
 
 # .deb package options
-set (CPACK_DEBIAN_PACKAGE_VERSION "${CPACK_PROJECT_VERSION}")
-set (CPACK_DEBIAN_PACKAGE_HOMEPAGE ${PACKAGE_URL})
-set (CPACK_DEBIAN_PACKAGE_MAINTAINER "Andreas Butti <andreasbutti@gmail.com>")
```

Comment:
These lines should not be removed. We should update the maintainer though as Andreas has paused his work on Xournal++ indefinitely for now.

Reply:
I don't know why I romoved them, but I think those are set by the project variables.

Reply:
`CPACK_DEBIAN_PACKAGE_VERSION` will be set by default.
`CPACK_DEBIAN_PACKAGE_HOMEPAGE` will be set by default.
`CPACK_DEBIAN_PACKAGE_MAINTAINER` is set to `CPACK_PACKAGE_CONTACT`, but that variable does not exist,
so I propose we chose a new contact and set `CPACK_PACKAGE_CONTACT` to it.

Reply:
Okay then lets set the contact and we are good. It has to be in the previously used format to be supported by Debian.

Reply:
Still missing the update to the maintainer